jakarta-jsonp/src/main/resources/META-INF/LICENSE

```diff
@@ -0,0 +1,8 @@
+This copy of Jackson JSON processor streaming parser/generator is licensed under the
```

Comment:
Should be modified to refer to actual component, I think...

Reply:
updated the names in the licenses